src/load-data.js

```diff
@@ -2,7 +2,7 @@
 
 const TILL_KEY = 'SPLITIO.splits.till'
 
-export default function loadDataIntoLocalStorage ({ serializedData = {}, userId = '' }, windowLocalStorage = {}) {
+export default function loadDataIntoLocalStorage ({ serializedData = {}, userId = '' }, windowLocalStorage = window.localStorage) {
```

Comment:
why this change?

Reply:
We are using dependency injection (of `windowLocalStorage` param) for testing purposes. We shouldn't require clients to have to pass in `windowLocalStorage: window.localStorage` param.